user_tools/src/spark_rapids_dataproc_tools/resources/profiling-conf.yaml

```diff
@@ -53,4 +53,4 @@ local:
 platform:
   workDir: rapids_user_tools_profiling
   outputDir: profiling_tool_output
-  cleanUp: true
\ No newline at end of file
+  cleanUp: true
```

Comment:
Not sure why this line has diff.
May from different edit tool. I used vim.

Reply:
previous version missed linefeed at end of file, which is auto fixed by vim 👍